Services/Curl.php

```diff
@@ -95,7 +95,9 @@ public function sendRequest($url, $method, array $options = array(), $payload =
         $curlMetaData = (object) curl_getinfo($this->curl);
 
         $this->curlOptionsHandler->reset();
-        function_exists(curl_reset($this->curl)) ? curl_reset($this->curl) : curl_setopt_array($this->curl, $this->curlOptionsHandler->getOptions());
+        if (function_exists('curl_reset')) {
```

Comment:
curl_reset is used for resetting the curl to the default state, so curl_setopt_array() is used for resetting the custom curl options of the current curl object (kind of a soft reset). In my opinion there is no need to reset other stuff than the custom options to the default ones, if the curl_reset function does not exist.. If you see another part that needs to be reseted let us know...